application/controllers/home.php

```diff
@@ -0,0 +1,36 @@
+<?php
+
+/**
+ * 
+ * home is a controller
+ * 
+ */
+
+class home
+{
```

Comment:
As per standards the curly braces `{`shouldn't start from the next line.It should be `class home {`

Reply:
Since I have used this pattern in each file, should I still change it?